skymarshal/skycmd/saml_flags.go

```diff
@@ -0,0 +1,82 @@
+package skycmd
+
+import (
+	"encoding/json"
+	"errors"
+
+	"github.com/concourse/dex/connector/saml"
+	"github.com/concourse/flag"
+	multierror "github.com/hashicorp/go-multierror"
+)
+
+func init() {
+	RegisterConnector(&Connector{
+		id:         "saml",
+		config:     &SAMLFlags{},
+		teamConfig: &SAMLTeamFlags{},
+	})
+}
+
+type SAMLFlags struct {
+	DisplayName        string      `long:"display-name" description:"The auth provider name displayed to users on the login page"`
+	SsoURL             string      `long:"sso-url" description:"(Required) SSO URL used for POST value"`
```

Comment:
I notice that dex also supports `EntityIssuer` and `SSOIssuer` fields - not sure what they are for. Is there a reason you didn't include them?

Reply:
Thanks for pointing this out, I didn't include them as they were not required when I tested the okta integration, but for generic SAML use-cases someone might require these flags, so I suppose we can pass them through to dex